src/bot.js

```diff
@@ -206,6 +206,122 @@ bot.on('message', async (message) => {
             }
         }
 
+        if (command === 'play' && args[0] === 'tf' && args[1] === 'competitive' && args.length === 2){
+
+            let triviaData = {};
+
+            let response;
```

Comment:
We should probably refactor this to not have to do that additional loop like I was doing previously, but we can worry about that later